.github/release-drafter.yml

```diff
@@ -1,5 +1,5 @@
-name-template: 'kudo-v$NEXT_PATCH_VERSION'
-tag-template: 'v$NEXT_PATCH_VERSION'
+name-template: 'Draft'
```

Comment:
does it make sense to assume we will have more than 1 draft?
1. 1 for master (which will be $NEXT_MINOR_VERSION)
2. 1 or more for release branches (i.e., `releases/0.10` which will be $NEXT_PATCH_VERSION)

Reply:
Yeah, that may be possible. We'd have to look into the when we're doing patch releases. At the moment the release drafter only watches the master-branch, so we only get the one draft, but we could set it up for release-branches as well.

I've changed the `tag-template` a little bit in this PR, as i'm not totally sure how GitHub works in this regard. If we use the actual tag template for the release, it might be that go-releaser wouldn't update the body of the ReleasePage and just append the released files. 

I think we have to play around a little bit with that in the next releases.